bindings/pyroot/pythonizations/python/ROOT/__init__.py

```diff
@@ -8,16 +8,23 @@
 # For the list of contributors see $ROOTSYS/README/CREDITS.                    #
 ################################################################################
 
-from os import environ
+import os
+import platform
 
 # Prevent cppyy's check for the PCH
-environ["CLING_STANDARD_PCH"] = "none"
+os.environ["CLING_STANDARD_PCH"] = "none"
 
 # Prevent cppyy's check for extra header directory
-environ["CPPYY_API_PATH"] = "none"
+os.environ["CPPYY_API_PATH"] = "none"
 
 # Prevent cppyy from filtering ROOT libraries
-environ["CPPYY_NO_ROOT_FILTER"] = "1"
+os.environ["CPPYY_NO_ROOT_FILTER"] = "1"
+
+# For macOS, need to load cppyy backend library with absolute path (see also PR #9725)
+if latform.system() == "Darwin":
```

Comment:
```suggestion
if platform.system() == "Darwin":
```

Reply:
Ah, damn :laughing: Thanks for the comment